src/sql/Sql.cpp

```diff
@@ -5,4 +5,43 @@ namespace bakcyl::sql {
     Sql::Sql() {
 	
     };
+    
+    std::vector<Product> Sql::getAllProducts() const {
+        std::vector<Product> products;
+
+        {
+            QSqlDatabase database = QSqlDatabase::addDatabase("");
```

Comment:
Czy to jest zamierzone, żeby przy każdej operacji otwierać i zamykać połączenie do bazy danych? Może lepiej byłoby je trzymać otwarte

Reply:
Być może źle napisałem coś w cmake'ach.

Reply:
https://github.com/mnurzyns/BakcylProgramowania2021/tree/dev/sql/database-connection

W tym tylko test psuje kompilacje

Reply:
Prawdopodobnie udało mi się rozwiązać problem z kompilacją

Reply:
Tak po podlinkowaniu QtSql w cmake/add_ut.cmake, z którego korzystają testy wszystko się ładnie kompiluje nawet z includem w .hpp